src/sentry/api/endpoints/user_authenticator_details.py

```diff
@@ -1,28 +1,110 @@
 from __future__ import absolute_import
 
 from django.db import transaction
+from rest_framework import status
+from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
 
-from sentry.api.bases.user import UserEndpoint
-from sentry.api.permissions import SuperuserPermission
+from sentry.api.bases.user import UserEndpoint, UserPermission
+from sentry.api.decorators import sudo_required
+from sentry.api.serializers import serialize
 from sentry.models import Authenticator
 from sentry.security import capture_security_activity
 
 
 class UserAuthenticatorDetailsEndpoint(UserEndpoint):
-    # XXX(dcramer): this requires superuser until we sort out how it will be
-    # used from the React app (which will require some kind of double
-    # verification)
-    permission_classes = (SuperuserPermission, )
+    permission_classes = (IsAuthenticated, UserPermission, )
```

Comment:
`UserPermission` handles superuser perms if `user != request.user`. (There's an outstanding PR which I'll take over and finish to gain authorization for superuser).

Reply:
Actually this isn't even necessary because this is a `UserEndpoint`